project/Dependencies.scala

```diff
@@ -383,7 +383,8 @@ object Dependencies {
 
   val UnixDomainSocket = Seq(
     libraryDependencies ++= Seq(
-      "com.github.jnr" % "jnr-unixsocket" % "0.19" // BSD/ApacheV2/CPL/MIT as per https://github.com/akka/alpakka/issues/620#issuecomment-348727265
+      "com.github.jnr" % "jffi" % "1.2.17" classifier "complete", // ApacheV2
```

Comment:
This took me a very long time to work out - you need the `complete` jar given the way sbt/ivy downloads artifacts.